packages/sirius-web/backend/sirius-web-sample-application/src/main/java/org/eclipse/sirius/web/sample/configuration/DomainPropertiesConfigurer.java

```diff
@@ -93,14 +109,35 @@ private FormDescription getAttributeDetails() {
         page.setLabelExpression("aql:self.name + ': ' + self.getDataType().capitalize()");
         form.getPages().add(page);
         page.getGroups().add(this.createGroup());
+
+        PageDescription entityPage = FormFactory.eINSTANCE.createPageDescription();
+        entityPage.setDomainType("domain::Entity");
```

Comment:
I don't think, this could work, you're trying to add a page on domain Entity in a form on domain Attribute.

Reply:
This is okay, we discussed together this topic.